beacon-chain/rpc/eth/beacon/handlers_pool.go

```diff
@@ -320,6 +320,13 @@ func (s *Server) handleAttestationsElectra(
 	}
 
 	for i, singleAtt := range validAttestations {
+		s.OperationNotifier.OperationFeed().Send(&feed.Event{
```

Comment:
why did we need to move this?

Reply:
So that we send a single attestation and not an Electra one